contentful_management/errors.py

```diff
@@ -87,7 +87,7 @@ class ServiceUnavailableError(HTTPError):
 
 
 def get_error(response):
-    """Gets Error by HTTP Status Code"""
+    """Gets Error by HTTP status code."""
```

Comment:
I see that you usually split the comments in multiple lines, like
``` python
"""
Base HTTP error class
"""
```

even when they're a single line, but you don't do it here 🤔 I think that for consistency you should probably choose one of those two alternatives, and stick with it 🙂